src/main/java/aquality/selenium/forms/Form.java

```diff
@@ -7,8 +7,10 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.Dimension;
 
-
-public abstract class Form {
+/**
+ * Defines base class for any UI form.
+ */
+public abstract class Form extends FormParent {
```

Comment:
What is the FormParent? Why do we need it?

Reply:
FormParent looks the same as IElement. For me it looks quite weird and should be simplified.

Reply:
As stated in src/main/java/aquality/selenium/forms/FormParent.java annotation, IParent is  
* Abstraction to support finding elements relative to form element.
 * Implements same methods as defined for 
 * @see IElement, but with protected access to them.

It's just a helper class in order not to make a mess out of Form class.

Reply:
Refactored, removed FormParent